pip_audit/format/columns.py

```diff
@@ -0,0 +1,60 @@
+from itertools import zip_longest
+from typing import Any, Dict, Iterable, List, Tuple
+
+import pip_audit.service as service
+
+from .interface import VulnerabilityFormat
+
+
+def tabulate(rows: Iterable[Iterable[Any]]) -> Tuple[List[str], List[int]]:
```

Comment:
I didn't think this was worth bringing in a dependency so I looked at how `pip list` does it and ripped the implementation out.

Reply:
Works for me! cc @di for opinions.

If we do end up wanting to offload this to a dependency, https://github.com/astanin/python-tabulate is probably a good option.

Reply:
This seems fine to me!